docs/idle-crafting.rst

```diff
@@ -0,0 +1,35 @@
+idle-crafting
+=============
+
+.. dfhack-tool::
+    :summary: Allow dwarfs to automatically satisfy their need to craft objects.
+    :tags: fort gameplay
+
+This script allows dwarves to automatically satisfy their crafting needs. The
+script is configured through an overlay that is added to the main page of
+craftsdwarf's workshops, as described below.
+
+Usage
+-----
+
+::
+
+    idle-crafting status
```

Comment:
also needs an `enable idle-crafting` line to indicate that it is enableable. no need to document `disable idle-crafting`. That's assumed.

Reply:
I'm not sure what the "enabled" contract for this tool should be. Currently, it just enables itself when a workshop is designated for idle crafting, and it disables itself once there are no more workshops designated for idle crafting.

Reply:
I came to the conclusion that `enable` doesn't make my sense for this tool. Thus `enable idle-crafting` will simply output an explanatory message. However, `disable idle-crafting` does make sense and is now implemented as forbidding idle crafting at all workshops (same as `idle-crafting disable`).